packages/core/src/html/headerProcessor.ts

```diff
@@ -12,22 +12,26 @@ const _ = {
 /*
  * h1 - h6
  */
-export function setHeadingId(node: MbNode, config: NodeProcessorConfig) {
+export function setHeadingId(node: MbNode,
```

Comment:
Hmmm this name is rather misleading because the headingId is only set IF not called from siteLinkManager. And it also returns the value - doesn't just set

Reply:
Renamed to "processHeadingId" and improved method documentation.